server/src/handlers/handlers.ts

```diff
@@ -364,3 +367,31 @@ export function handleCrash({ status, result }: socket.FlixResponse) {
     ],
   })
 }
+
+/**
+ * Handle document formatting requests.
+ *
+ * @param params - The document formatting parameters.
+ * @returns A promise that resolves to an array of text edits.
+ */
+export const handleDocumentFormatting = (params: DocumentFormattingParams): Thenable<TextEdit[]> => {
```

Comment:
I would move this a bit higher up in the file, so its next to related functionality.